tests/ember-intl/tests/integration/helpers/format-date-range/input-is-a-string-test.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-irregular-whitespace, qunit/no-commented-tests */
```

Comment:
**TODO**

Uncomment these tests after https://github.com/formatjs/formatjs/pull/4784 is merged and released.

Reply:
Another option would be to keep the tests uncommented and use `test.skip()` instead of `test()` (just in case you didn't know 😊).

Reply:
I do. But the unsupported string case causes Glint to error and it's not easy to add a `@glint-expect-error` in a multi-line invocation.